cypress/tests/add-capacity.spec.ts

```diff
@@ -0,0 +1,154 @@
+      const storageCluster = JSON.parse(res.stdout);
+      _.set(initialState, 'storageCluster', storageCluster);
+    });
+    cy.exec('oc get cephCluster ocs-storagecluster-cephcluster -n openshift-storage -o json').then(
+      (res) => {
+        const cephCluster = JSON.parse(res.stdout);
+        _.set(initialState, 'cephCluster', cephCluster);
+
+        cy.log('Check if ceph cluster is healthy before expansion');
+        expect(cephCluster.status.ceph.health).not.to.equal(CLUSTER_STATUS.HEALTH_ERROR);
+      },
+    );
+    cy.exec(
+      `oc -n openshift-storage rsh $(oc get po -n openshift-storage | grep ceph-operator | awk '{print$1}') ceph --conf=/var/lib/rook/openshift-storage/openshift-storage.config osd tree --format=json`,
+      { timeout: 120000 },
+    ).then((res) => {
+      const osdTree = JSON.parse(res.stdout);
+      _.set(initialState, 'osdTree', osdTree);
+
+      const formattedOSDTree = createOSDTreeMap(osdTree.nodes);
+      _.set(initialState, 'formattedOSDTree', formattedOSDTree);
+
+      const osdIDs = getIds(osdTree.nodes, 'osd');
+      _.set(initialState, 'osdIDs', osdIDs);
+    });
+    cy.exec('oc get po -n openshift-storage -o json').then((res) => {
+      const pods = JSON.parse(res.stdout);
+      _.set(initialState, 'pods', pods);
+
+      cy.clickNavLink(['Storage', 'OpenShift Data Foundation']);
+      cy.contains('Storage Systems').click();
+      cy.byLegacyTestID('kebab-button').click();
+      cy.byTestActionID('Add Capacity').click();
+      modal.shouldBeOpened();
+
+      const initialCapcity =
+        SIZE_MAP[
+        initialState.storageCluster?.spec?.storageDeviceSets?.[0]?.dataPVCTemplate?.spec
+          ?.resources?.requests?.storage
+        ];
+      cy.byLegacyTestID('requestSize').should('have.value', String(initialCapcity));
+      cy.byTestID('provisioned-capacity').contains(
+        `${String((initialCapcity * 3).toFixed(2))} TiB`,
+      );
+      cy.byTestID('add-cap-sc-dropdown', { timeout: 10000 }).should('be.visible');
+      modal.submit();
+      modal.shouldBeClosed();
+
+      cy.clickNavLink(['Operators', 'Installed Operators']);
+      cy.byLegacyTestID('item-filter').type("Openshift Data Foundation");
+      cy.byTestRows('resource-row').get('td').first().click();
+      cy.byLegacyTestID('horizontal-link-Storage System').click();
+      cy.contains(OCS_SS).click();
+      cy.contains('Resources').click();
+      cy.byTestOperandLink(OCS_SC).click();
+      // Wait for the storage cluster to reach Ready
+      // Storage Cluster CR flickers so wait for 10 seconds
+      // Disablng until ocs-operator fixes above issue
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(10000);
+      cy.byTestID('resource-status').contains('Ready', { timeout: 900000 });
+    });
+    cy.exec('oc get storagecluster ocs-storagecluster -n openshift-storage -o json').then((res) => {
```

Comment:
```suggestion
    cy.exec(`oc get storagecluster ${OCS_SC} -n openshift-storage -o json`).then((res) => {
```

since we have already declared constants